app/initializers/ember-cli-mirage.js

```diff
@@ -1,37 +1,45 @@
-import Ember from 'ember';
-import readModules from 'ember-cli-mirage/utils/read-modules';
 import ENV from '../config/environment';
 import baseConfig, { testConfig } from '../mirage/config';
-import Server from 'ember-cli-mirage/server';
-import _assign from 'lodash/assign';
-
-const {
-  getWithDefault
-} = Ember;
-
+import getRfc232TestContext from 'ember-cli-mirage/get-rfc232-test-context';
+import newStartMirage from 'ember-cli-mirage/start-mirage';
```

Comment:
I'd vaguely prefer a different variable name here (I'm always leery of using "new" or "legacy" in variable names because they loose meaning over time), though I don't have a better suggestion...

Reply:
Yeah, I totally agree -- over in my `ember-cli-page-object` PR, I wrote (mostly in comments) "new test API" and then went back and changed it to "rfc232/rfc268 test API" like a thousand times 😄 

Maybe `realStartMirage` or `startMirageImpl`?